packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -121,9 +121,11 @@ const AccountTransferForm = observer(
             resetConverter,
         } = crypto_fiat_converter;
 
-        // TODO: Replace 'login' with 'open_order_position_status' once the flag is available in the BE response and in type TSocketResponseData<"mt5_login_list">
-        const { is_flag_present, flag_value } = useIsMt5LoginListStatusPresent('login', selected_to.value ?? '');
-        const is_open_order_position_status_present = is_flag_present && !!flag_value;
+        const { is_flag_present, flag_value } = useIsMt5LoginListStatusPresent(
+            'open_order_position_status',
+            selected_to.value ?? ''
+        );
+        const is_open_order_position_status_present = is_flag_present && (flag_value === 0 || flag_value === 1);
```

Comment:
Can u add a note on the available `flag_value` please? It will help in understanding the code better